common/recommender/Baseline.js

```diff
@@ -42,13 +42,13 @@ class Baseline {
 
     const age = this.normalizeTimestamp(entry.lastVisitDate);
     const imageCount = entry.images ? entry.images.length : 0;
-    const isBookmarked = !!entry.bookmarkDateCreated;
+    const bookmarkAge = entry.bookmarkDateCreated ? Math.max(1, Date.now() - entry.bookmarkDateCreated) : 0;
```

Comment:
For the better readablity, please add a comment for `bookmarkAge`. For example, calculate the age in milliseconds (minimal as 1) if it's a bookmark, otherwise using 0 for non-bookmark links"